bokeh/plotting.py

```diff
@@ -410,13 +410,13 @@ def output_file(filename, title="Bokeh Plot", autosave=False, mode="inline", roo
         print("Session output file '%s' already exists, will be overwritten." % filename)
 
 
-def show(obj=None, browser=None, new="tab", url=None):
-    """ 'shows' a plot object or the current plot, by auto-raising the window or tab
-    displaying the current plot (for file/server output modes) or displaying
-    it in an output cell (IPython notebook).
+def show(obj, browser=None, new="tab", url=None):
+    """ 'shows' a plot object, by auto-raising the window or tab displaying the
+    current plot (for file/server output modes) or displaying it in an output
```

Comment:
would suggest:
```
def show(obj, browser=None, new="tab", url=None):
    """ Immediately display a plot object.

    In an IPython/Jupyter notebook, the output is displayed in an output cell. Otherwise, 
    a browser window or tab is autoraised to display the plot object. 

```